Реализованы методы получения списка направлений по id пациента, установки статуса анализу, установки статуса направлению

hacksocto opened the pull request:
Реализован метод получения списка направлений по id пациента
Реализован метод установки статуса анализу
Реализован метод установки статуса направлению

hacksocto committed:
Реализован метод получения списка направлений по id пациента
Реализован метод установки статуса анализу
Реализован метод установки статуса направлению

JulianaOsi commented on pkg/store/analysis.go:
```diff
@@ -48,6 +48,42 @@ func (s *Store) GetAnalysisByDirectionId(ctx context.Context, directionId string
 
+func (s *Store) SetAnalysisChecked(ctx context.Context, analysisId string) error {
+	sql, _, err := goqu.Update("direction_analysis").
+		Set(goqu.Record{"is_checked": "True"}).
```
во входные параметры должна передаваться булева переменная isChecked, значение которой и нужно установить колонке is_checked.

JulianaOsi commented on pkg/store/analysis.go:
```diff
@@ -48,6 +48,42 @@ func (s *Store) GetAnalysisByDirectionId(ctx context.Context, directionId string
+	return nil
+}
+
+func (s *Store) SetAnalysisUnChecked(ctx context.Context, analysisId string) error {
```
не нужно делать два метода для установки значения в одной и той же колонке

hacksocto committed:
Реализован метод получения списка направлений по id пациента
Реализован метод установки статуса анализу
Реализован метод установки статуса направлению
